portal/code/cx-portal-shared-components/src/components/basic/IconButton/index.tsx

```diff
@@ -0,0 +1,19 @@
+import MuiIconButton, {
+  IconButtonProps as MuiIconButtonProps,
+} from '@mui/material/IconButton'
+
+interface IconButtonProps extends Omit<MuiIconButtonProps, 'color' | 'size'> {
+  color?: 'primary' | 'secondary'
+  size?: 'medium' | 'small'
+  variant?: 'outlined'
+}
+
+export const IconButton = ({
+  size = 'medium',
+  variant,
+  ...props
+}: IconButtonProps) => {
+  const outlinedStyle = variant === 'outlined' ? { border: '2px solid' } : {}
```

Comment:
Why has this to be hard coded here? If other components are using the variant 'outlined' we have to copy it. Prefer this in a global style or theme